src/OrchardCore/OrchardCore.Abstractions/Modules/Builder/OrchardCoreBuilder.cs

```diff
@@ -56,6 +57,26 @@ public OrchardCoreBuilder ConfigureServices(Action<IServiceCollection> configure
             return ConfigureServices((s, sp) => configure(s), order);
         }
 
+        /// <summary>
+        /// This async method gets called for each tenant. Use this method to initialize container services.
```

Comment:
```suggestion
        /// Use this method to initialize tenant services asynchronously.
```

Reply:
Yes, I always hesitate to be consistent with what is already there to not have to do too much updates, for example here in the same file we have the following comments including this one.

        /// This method gets called for each tenant. Use this method to add services to the container.
        /// This async method gets called for each tenant. Use this method to initialize container services.
        /// This method gets called for each tenant. Use this method to configure the request's pipeline.

If I apply the suggestion we would have

        /// This method gets called for each tenant. Use this method to add services to the container.
        /// Use this method to initialize tenant services asynchronously.
        /// This method gets called for each tenant. Use this method to configure the request's pipeline.

That's why I hesiated between keeping the same kind of comments or updating all of them ;)